Library/Homebrew/brew.sh

```diff
@@ -6,7 +6,10 @@ case "$HOMEBREW_SYSTEM" in
 esac
 
 # Force UTF-8 to avoid encoding issues for users with broken locale settings.
-if [[ "$(locale charmap 2>/dev/null)" != "UTF-8" ]]
+if ! which locale &>/dev/null
```

Comment:
```suggestion
if ! command -v locale >/dev/null
```
`command` is a shell builtin, so preferred over `which`.

Reply:
Could also consider just calling this but hiding errors.